test/functorch/test_aotdispatch.py

```diff
@@ -364,6 +364,16 @@ def forward(self, primals_1):
     mul_1 = torch.ops.aten.mul.Tensor(mul, 3)
     return [mul, mul_1]""")
 
+    def test_input_mutation_simple_with_none(self):
+        def f(a, b):
+            return a * 3
+        inp = [torch.ones(3, 3, requires_grad=True), None]
```

Comment:
Could you add other varieties of non-tensor inputs as well? e.g. list<int> , int ?